frontend/src/app/modules/authentication/registration/registration.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractControl, FormBuilder, FormGroup, ValidatorFn, Validators } from '@angular/forms';
+
+import { UserRegisterDto } from '../../../models/auth/user-register-dto';
+
+@Component({
+    selector: 'app-registration',
+    templateUrl: './registration.component.html',
+    styleUrls: ['./registration.component.sass'],
+})
+export class RegistrationComponent implements OnInit {
+    registerForm: FormGroup = new FormGroup({});
+
+    showPassword = false;
+
+    showConfirmPassword = false;
+
+    private fb: FormBuilder;
```

Comment:
why all fields and methods does not have explicit access modifier (private/public), whereas fb field has private modifier. Maybe it is better to write all access modifiers explicitly as we do in C#

Reply:
yea, good point